e2e/tests/test_cases/single_replica_node_down.robot

```diff
@@ -0,0 +1,136 @@
+*** Settings ***
+Documentation    Single replica node down
+...              https://longhorn.github.io/longhorn-tests/manual/pre-release/node-not-ready/node-down/single-replica-node-down/
+
+Test Tags    manual_test_case
+
+Resource    ../keywords/common.resource
+Resource    ../keywords/deployment.resource
+Resource    ../keywords/persistentvolumeclaim.resource
+Resource    ../keywords/recurringjob.resource
+Resource    ../keywords/statefulset.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/workload.resource
+Resource    ../keywords/host.resource
+
+Library    OperatingSystem
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources include off nodes
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+
+*** Test Cases ***
+Single Replica Node Down Deletion Policy do-nothing With RWO Volume Replica Locate On Replica Node
+    Given Set setting node-down-pod-deletion-policy to do-nothing
+    When Create persistentvolumeclaim 0 using RWO volume
+    And Create deployment 0 with persistentvolumeclaim 0
+    And Write 100 MB data to file data in deployment 0
+
+    # Delete replicas to have the volume with its only replica located on different nodes.
+    And Update volume of deployment 0 replica count to 1
+    And Delete replica of deployment 0 volume on replica node
+    And Delete replica of deployment 0 volume on volume node
+    And Power off volume node of deployment 0
+    Then Wait for volume of deployment 0 attaching
```

Comment:
The original step is `Volume will fail to attach since volume is not ready (i.e remains faulted, since single replica is on downed node).`

But `Then Wait for volume of deployment 0 attaching` doesn't seem to indicate the volume is stuck in a state. WDYT?

Reply:
Updated `Then Wait for volume of deployment 0 stuck in state attaching`